src/service/resources-impl.js

```diff
@@ -222,6 +226,12 @@ export class Resources {
       this.lastScrollTime_ = Date.now();
     });
 
+    if (this.useLayers_) {
+      Services.layersForDoc(this.ampdoc).onScroll((/* elements */) => {
```

Comment:
Could be a bit chatty, no?

Reply:
Yes. The aim for a later PR is to pass in `elements`, which is an array of all the elements that scrolled. This will allow us to ignore resources in unscrolled layers.

If you're suggesting a throttle, I didn't do that because `#schedulePass` is itself a throttle.